xija/component/heat.py

```diff
@@ -640,6 +654,21 @@ def dvals(self):
             else:
                 self.calc_earth_vis_from_taco(ephems, q_atts)
 
+            # This next bit optionally checks to see if the solar ephemeris
+            # was passed in, and if it was it computes the fraction of the
+            # Earth's surface that is illuminated by the Sun. 
```

Comment:
Include a link or reference to the TWG meeting where you presented this, so we can get back to that nice hand-drawn picture of the math.